rpm/util-linux.spec

```diff
@@ -198,6 +199,24 @@ Obsoletes: %{name}-docs
 %description doc
 Man and info pages for %{name}.
 
+%package systemd-fstrim
+Summary:   Systemd unit files for %{name}
+Group:     System
```

Comment:
Please, drop `Group:` as we don't use it any more.

Reply:
Please @okodron, do implement these cosmetic changes, if you have any additional ones:
- This PR is explicitly marked as "maintainers may alter it".
- As a community contributor, I am doing this in my spare time, you in your paid time.
- I already contributed the suggestion (2 years ago), plus now an implementation (by @nephros), which is tested (by him and me).
- As a community contributor, Jolla's non-public style-guides and its changes ("we don't use this anymore") are unknown to me.

Hence feel free to perform further adaptions, as long as the intended functionality is retained.
Note that I am sure willing to continue discussing technical aspects of this PR.

Reply:
@okodron, when thoroughly reviewing your request in the spec file, it becomes obvious that the principal package and every sub-package carries a `Group:` statement.  Hence defining the apropiate Groups for the two newly added sub-packages is aligned to the rest of the spec file style.  If you want to change the current style, you may do that, but that would definitely expand the scope of this PR, hence I suggest to perform this change in a separate PR, because it is unrelated to the content and intention of this PR, plus such a style change should be properly and visibly (hence separately) documented.

Reply:
Ok, it can be done in a separate MR cause it's not strictly related to these changes.